src/npc_class.cpp

```diff
@@ -267,7 +267,7 @@ distribution distribution::dice_roll( int sides, int size )
     } );
 }
 
-distribution distribution::operator+( const distribution &other ) const
+distribution distribution::sum( const distribution &other ) const
```

Comment:
So here `sum` means "take this, add the other parameter to it and return it, but keep this unchanged!"

In `resistances::sum`, it means `add the parameter to this, return reference to this".

How is that inconsistency any better than `operator+` / `operator+=`? At least there I know that `+=` mutates the left side, whereas `+` does not affect either operand.

Also: how is a function named `sum` any better than using the addition (also called summation) operator?